docs/manage/_command_line.rst

```diff
@@ -155,12 +155,48 @@ If you need to restore a backup version prior to the latest one, you must specif
   This command is not intended for replication across different devices, but **only** for restoring on a single device from a local backup of the database.
 
 
+Change the Location of Kolibri Content Files
+--------------------------------------------
+
+Kolibri content channels may occupy a considerable amount of hard disk space over time. If you have concerns about running out of storage on your device, you can move just the Kolibri **content files** on another drive.
+
+.. tip::
+  If you have both SSD disk and HDD disk available on your device, it is recommended to install Kolibri on the SSD drive to allow faster access to the database, and move just the content file to the HDD drive.
+
+To move the Kolibri content folders to another location, follow these steps.
+
+1. Stop Kolibri.
+
+  .. code-block:: bash
+
+    kolibri stop
+
+
+2. Create a new folder that will contain all the content files and resources on the destination drive.
+
+  .. code-block:: bash
+
+    kolibri manage content movedirectory <destination>
+
+
+  For example, if you created a new folder ``KolibriContent`` on the drive ``F:``, run this command.
+
+  .. code-block:: bash
+
+    kolibri manage content movedirectory F:\KolibriContent
```

Comment:
The commands look linux-specific, but this is a windows-looking path...

Reply:
:+1: I think for consistency, we should keep all generic examples with POSIX paths. Another aspect is that we might not want to encourage Windows for deployments, a good way to show that is to avoid it in examples (but of course to clarify to Windows users how they need to alter commands).

    kolibri manage content movedirectory /mnt/my_external_drive/kolibri_content

Reply:
I'd love if less deployments were done on Windows, but I'm skeptical about the strategy of not documenting Windows-specific steps as a means to achieve it. Knowing our target user base, I'm less worried about Linux/macOS Terminal users figuring out the way to write POSIX paths by looking at Windows-looking one, than viceversa. 

I added both examples for now.